client/galaxy/scripts/mvc/dataset/dataset-edit-attributes.js

```diff
@@ -0,0 +1,261 @@
+        initialize: function( options ) {
+            options = options || {};
+            options.dataset_id = options.dataset_id;
+        }
+    });
+
+    /** Dataset edit attributes view */
+    var View = Backbone.View.extend({
+        initialize: function() {
+            this.setElement( '<div/>' );
+            this.model = new Model( { 'dataset_id': Galaxy.params.dataset_id } );
+            this.render();
+        },
+
+        // Fetch data for the selected dataset and 
+        // build tabs for editing its attributes
+        render: function() {
+            var url = Galaxy.root + 'dataset/edit',
+                self = this;
+            Utils.get({
+                url     : url,
+                data    : { 'dataset_id' : self.model.get( 'dataset_id' ) },
+                success : function( response ) {
+                   self.render_attribute_page( self, response );
+                },
+                error   : function( response ) {
+                    var error_response = {
+                        'status': 'error',
+                        'message': 'Error occured while loading the dataset.',
+                        'persistent': true,
+                        'cls': 'errormessage'
+                    };
+                    self.display_message( self, error_response, self.$el.find( '.response-message' ) );
+                }
+            });
+        },
+
+        /** Render all the tabs view */
+        render_attribute_page: function( self, response ) {
+            var message = {
+                'message'     : response.message,
+                'status'      : response.status,
+                'persistent'  : true,
+                'cls'         : response.status + 'message'
+            };
+            self.$el.empty().append( self._templateHeader() );
+            self.display_message( self, message, self.$el.find( '.response-message' ) );
+            // Create all tabs
+            self.create_tabs( self, response, self.$el.find( '.edit-attr' ) );
+        },
+
+        /** Perform AJAX post call */
+        call_ajax: function( self, data, tab_name ) {
+            var post_url = Galaxy.root + 'dataset/edit';
+            $.ajax({
+                type: "PUT",
+                url: post_url,
+                data: data,
+                success: function( response ) {
+                    self.render_attribute_page( self, response );
+                    self.reload_history();
+                },
+                error   : function( response ) {
+                    var error_response = {
+                        'status': 'error',
+                        'message': 'Error occured while saving. Please fill all the required fields and try again.',
+                        'persistent': true,
+                        'cls': 'errormessage'
+                    };
+                    self.display_message( self, error_response, self.$el.find( '.response-message' ) );
```

Comment:
Parsing `self` as first parameter is not necessary. I made a similar comment in the other PR. Same goes for `self.$el.find('')` vs `self.$('')`.